packages/driver/src/errors/base.ts

```diff
@@ -65,7 +64,9 @@ function tryParseInt(val: any) {
   if (val instanceof Uint8Array) {
     try {
       return parseInt(utf8Decoder.decode(val), 10);
-    } catch {}
+    } catch (err) {
+      console.error("Failed to parse integer value: ", val, err);
```

Comment:
If eslint is complaining about an empty block here, I'd rather not log an error here just to silence the linter.

Oh, actually, I think the fix here is to actually return `null` inside of this catch and remove it from the outer scope instead of silently swallowing the error, or loudly logging the error.

```suggestion
      return null;
```

Reply:
hmm well in this case null is returned if `if block` is not satisfied, so if we remove outer null then undefined will be returned in this case... I dont know why we explicitly return null but maybe there's a reason

Reply:
Well, when the `if` exits without having returned the `parseInt`, it will still return `null` here. So what we have here, logically is:

if "val" is an instance of "Uint8Array" and it we successfully parse an int from the UTF-8 buffer decoder with radix of 10, we return that parsed int.  otherwise, return `null`.

There isn't (currently) a flow where we return `undefined` here, and I'm not suggesting that either. Just:

```ts
function tryParseInt(val: any) {
  if (val instanceof Uint8Array) {
    try {
      return parseInt(utf8Decoder.decode(val), 10);
    } catch {
      return null;
    }
  }
  return null;
}
```

or if we want to do an early return:

```ts
function tryParseInt(val: any) {
  if (!(val instanceof Uint8Array) return null;
  try {
    return parseInt(utf8Decoder.decode(val), 10);
  } catch {
    return null;
  }
}
```

Reply:
ah sorry, I thought u said to delete "return null" from the outer scope (outside try/catch block) and return null only inside catch block...

Reply:
oh, I did say that originally! 🤦 glad you caught that, sorry for the confusion.